### PR TITLE
feat(apisix): add timezone and extraEnvVars

### PR DESCRIPTION
**Pull request by Gallardot**
You may need to set the timezone to be consistent with your local time zone, otherwise the apisix's logs may used to retrieve event maybe in wrong timezone. 

Also define `extraEnvVars` to allow users to extend additional environment variables

### CODE REVIEW

File: charts/apisix/values.yaml [hunk 1]
```diff
@@ -96,6 +96,16 @@ apisix:
   podAntiAffinity:
     enabled: false
 
+  # timezone is the timezone where apisix uses.
+  # For example: "UTC" or "Asia/Shanghai"
+  # This value will be set on apisix container's environment variable TZ.
+  # You may need to set the timezone to be consistent with your local time zone,
+  # otherwise the apisix's logs may used to retrieve event maybe in wrong timezone.
+  timezone: UTC
```
Comment (tokers) [inline charts/apisix/values.yaml:104]: Should add the documents to explain it.

File: charts/apisix/values.yaml [hunk 2]
```diff
@@ -96,6 +96,16 @@ apisix:
...
+
+  # extraEnvVars An array to add extra env vars
+  extraEnvVars: []
```
Comment (tokers) [inline charts/apisix/values.yaml:107]: The format for the key and value should be explained.

File: charts/apisix/values.yaml (hunk 1 above)
Comment (Gallardot) [reply charts/apisix/values.yaml:104]: > Should add the documents to explain it.

I also add this note on the `README` document.  I think this explanation document about `timezone` might be more appropriate in the [APISIX FAQ](https://apisix.apache.org/docs/apisix/FAQ).   @tokers  WDYT？

File: charts/apisix/values.yaml (hunk 2 above)
Comment (Gallardot) [reply charts/apisix/values.yaml:116]: Done! PTAL.

Comment (tokers) [reply charts/apisix/values.yaml:116]: Shall we also add it in the README?

File: charts/apisix/values.yaml (hunk 1 above)
Comment (tokers) [reply charts/apisix/values.yaml:104]: Agree with you, would you like to submit another PR to add it?

Comment (Gallardot) [reply charts/apisix/values.yaml:104]: > Agree with you, would you like to submit another PR to add it?

OK，I will submit another PR first and then deal with the current PR

**Comment by moonming** [comment]
the same as https://github.com/apache/apisix/issues/3039? we can not hard-code TZ

**Comment by tokers** [comment]
@Gallardot We may do not set the `TZ` if the value is empty.

**Review by tao12345666333 — APPROVED** [review]
LGTM

**Comment by Gallardot** [comment]
> @Gallardot We may do not set the `TZ` if the value is empty.

@tokers Done.   PTAL https://github.com/apache/apisix/pull/6705

File: charts/apisix/README.md [hunk 3]
```diff
@@ -76,9 +76,10 @@ The following tables lists the configurable parameters of the apisix chart and t
...
 | `apisix.luaModuleHook.configMapRef.mounts[].key` | Name of the ConfigMap key, for setting the mapping relationship between ConfigMap key and the lua module code path. | `""` |
 | `apisix.luaModuleHook.configMapRef.mounts[].path` | Filepath of the plugin code, for setting the mapping relationship between ConfigMap key and the lua module code path. | `""` |
-| `apisix.timezone` | The timezone where APISIX container uses. For example: UTC, Asia/Shanghai. This value will be set on apisix container's environment variable TZ. Sometimes may need to set the timezone to be consistent with local time zone, otherwise the apisix's logs may used to retrieve event maybe in wrong timezone.                | `UTC`|
-| `apisix.extraEnvVars` | APISIX container extra environment variables                   | `[]`|
-
+| `apisix.timezone` | The timezone where APISIX container uses. For example: UTC, Asia/Shanghai. This value will be set on apisix container's environment variable TZ. Sometimes may need to set the timezone to be consistent with local time zone, otherwise the apisix's logs may used to retrieve event maybe in wrong timezone.                | nil|
```
Comment (tokers) [inline charts/apisix/README.md:79]: The default value is not `nil`.

Comment (Gallardot) [reply charts/apisix/README.md]: Done. PTAL.

**Review by tokers — APPROVED** [review]

**Pull request merged**